src/graphics/graphics-device.js

```diff
@@ -82,6 +82,14 @@ class GraphicsDevice extends EventHandler {
      */
     supportsInstancing;
 
+    /**
+     * True if the device supports uniform buffers.
+     *
+     * @type {boolean}
+     * @ignore
+     */
+    supportsUniformBuffer = false;
```

Comment:
what's the reason to expose this flag? i thought we'd essentially simulate uniform buffers on platforms that don't support it, like webgl1? (so no need for user to know).

Reply:
it's a private flag (see the ignore tag) - it's used internally until all platforms have uniform buffer supported (in real or simulated way). This allows one platform to be implemented before the other one.